controllers/users.js

```diff
@@ -113,6 +114,39 @@ const getUsers = async (req, res) => {
       });
     }
 
+    if (req.query.profile) {
```

Comment:
if users sends ?profile=false, it will still pass this case. query params are strings value

Reply:
done